### DISCUSSION TITLE
Elasticsearch query supports cardinality aggregation

### DISCUSSION THREAD

**Discussion by kevisong** [grafana/grafana#41719] [Dashboards, Panels & Visualizations]
Add cardinality metric aggregation support for elasticsearch query.

[Cardinality aggregation](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-metrics-cardinality-aggregation.html#search-aggregations-metrics-cardinality-aggregation)

Currently only these are supported:

![image](https://user-images.githubusercontent.com/39701004/141935263-ba979c20-d037-4557-8b21-93b4ed1bbad9.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
